po/es.po

```diff
@@ -771,7 +771,7 @@ msgid ""
 "a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class="
 "\"gnucash\">GnuCash</span> wikis."
 msgstr ""
-"Otros recursos son las wikis de <span class=\"gnucash\">GnuCash</span> en <a "
+"Otros recursos son los wikis de <span class=\"gnucash\">GnuCash</span> en <a "
 "href=\"https://wiki.gnucash.org/wiki/\">Inglés</a> o <a href=\"https://wiki."
 "gnucash.org/wiki/De/\">Alemán</a>."
```

Comment:
@cyphra I just saw this after merging.
IMHO it would be more important to replace
`<a href=\"https://wiki.gnucash.org/wiki/De/\">Alemán</a>`
 by
`<a href=\"https://wiki.gnucash.org/wiki/Es/\">Español</a>`

Had you not seen the translators comment?